src/components/StatusMenu.js

```diff
@@ -271,7 +271,9 @@ const StatusMenu = ({ toggleStatus, isStatusOpen }) => {
               <div className="mb-8 inline-flex items-center">
                 <a
                   className="text-sm text-blue-tl"
-                  href="https://docs.trustedlogin.com/Vendor/intro">
+                  target={"_blank"}
```

Comment:
We are lacking accessibility affordances. For external links, please add `<span class="screen-reader-text">(This link opens in a new window.)</span>` to the items in this PR, then we can add for the rest of the links in the plugin.